src/flow_meter.py

```diff
@@ -54,7 +54,7 @@ def get_reading(self, duration):
         # dump what's already inside the buffer. this is only useful for low overhead
         #buffer = self.device.read_measured_value_buffer(Sfc5xxxScaling.USER_DEFINED)
         while len(reading) <= duration * 1000: # flow meter reads at 1kHz
-            buffer = self.device.read_measured_value_buffer(Sfc5xxxScaling.USER_DEFINED)
+            buffer = self.device.read_measured_value_buffer(Sfc5xxxScaling.USER_DEFINED, max_reads=2)
             #print(buffer.sampling_time) # the only "time" returned from read buffer command
             reading.extend(buffer.values)
         return reading
```

Comment:
return reading should return a numpy array here already.

Reply:
This is a choice of where to implement list-to-numpy array conversion. If doing it at a later time, we eliminate the need to import numpy in the flow meter section.